packages/aws-client/src/client.ts

```diff
@@ -7,8 +6,6 @@ import { getServiceIdentifer } from './utils';
 
 const getRegion = () => process.env.AWS_DEFAULT_REGION || process.env.AWS_REGION || 'us-east-1';
 
-AWS.config.setPromisesDependency(Promise);
```

Comment:
What did this do?

Reply:
AWS SDK v2 allowed you to configure the promise library it used and this is how you did it.